audinterface/core/process.py

```diff
@@ -36,6 +36,24 @@ class Process:
         resample: if ``True`` enforces given sampling rate by resampling
         channels: channel selection, see :func:`audresample.remix`
         mixdown: apply mono mix-down on selection
+        win_dur: window duration,
+            if processing should be applied on a sliding window.
+            If value is as a float or integer
+            it is treated as seconds.
+            To specify a unit provide as string,
+            e.g. ``'2ms'``.
+            To specify in samples provide as string without unit,
+            e.g. ``'2000'``
```

Comment:
I'm always not sure what the best way is to handle the following problem:
We now repeat how you can specify duration four times in the docstring (`win_dur`, `hop_dur`, `min_signal_dur`, `max_signal_dur`), and the same in other docstrings as well. So it would make sense to write it down once, maybe at the top of each docstring, or as a subsection in the user documentation.
On the other hand, if I look for a particular argument of the object, I will directly look for it and not read the top of the docstring. So from a user perspective the current "implemenation" of the docstring makes sense, but it is not the best solution for maintaining.

Do you have a preference (independent of that I would suggest to not handle a possible solution in this pull request, but go with your current solution)?

Reply:
Would it make sense to add `utils.to_timedelta()` to the API and then simply link to it in those places?

Reply:
Created https://github.com/audeering/audinterface/issues/59